djangocms_repeater/models.py

```diff
@@ -52,6 +53,9 @@ class Repeater(CMSPlugin):
     repeater_name = models.CharField(max_length=50)
     add_columns = models.BooleanField(default=False)
 
+    def __str__(self):
+        return '%s' % (self.repeater_name)
+
```

Comment:
I mean the `'%s" % (self.my_attribute)`, can't we just simply return `self.my_attribute` ?